entrypoint.sh

```diff
@@ -17,4 +17,17 @@ if [ -z "${SECRET_KEY_BASE}" ]; then
     exit 1
 fi
 
+command=$1
+
+case $command in
+  setup)
+    bundle exec rake railties:install:migrations db:create db:migrate
+    exit 0
+    ;;
+  upgrade)
+    bundle exec rake railties:install:migrations db:migrate
+    exit 0
+    ;;
+esac
+
```

Comment:
I like the simplicity of providing these commands. It is possible to run the following:

**setup**
```
docker run -i digitalocean/docker-shipit-engine:`tagname` "bundle exec rake railties:install:migrations db:create db:migrate"
```
**upgrade**
```
docker run -i digitalocean/docker-shipit-engine:`tagname` "bundle exec rake railties:install:migrations db:migrate"
```
But the commands simply these down to:

**setup**
```
docker run -i digitalocean/docker-shipit-engine:`tagname` setup
```
**upgrade**
```
docker run -i digitalocean/docker-shipit-engine:`tagname` upgrade
```